homeassistant/components/nest.py

```diff
@@ -88,8 +88,6 @@ def setup_nest(hass, nest, config, pin=None):
 
     _LOGGER.debug("proceeding with discovery")
     discovery.load_platform(hass, 'climate', DOMAIN, {}, config)
-    discovery.load_platform(hass, 'sensor', DOMAIN, {}, config)
```

Comment:
Nest Protects are _only_ sensors, so Nest Protect users would have to explicitly configure the sensor, which is what I was trying to avoid.